qiskit-ibm-runtime-mcp-server/src/qiskit_ibm_runtime_mcp_server/ibm_runtime.py

```diff
@@ -753,4 +755,216 @@ async def get_service_status() -> str:
         return f"IBM Quantum Service Status: {status_info}"
 
 
+@with_sync
+async def delete_saved_account(account_name: str = "") -> dict[str, Any]:
+    """
+    Delete a saved IBM Quantum account from disk.
+
+    **WARNING: DESTRUCTIVE OPERATION**
+    This permanently removes saved credentials on ~/.qiskit/qiskit-ibm.json.
+    The operation CANNOT be undone. Deleted credentials must be re-entered to restore access.
+
+    Args:
+        account_name: Name of the saved account to delete. If empty string (default),
+                     deletes the default account. Account names are typically in the
+                     format 'ibm_quantum_platform' or custom names set during save.
+
+    Returns:
+        Dictionary containing deletion status:
+        - On success: {"status": "success", "deleted": True, "message": "Account successfully deleted"}
+        - On failure: {"status": "error", "deleted": False, "error": "Account name not found or could not be deleted"}
+        - On error: {"status": "error", "deleted": False, "error": error_message}
+
+    """
+    global service
+
+    try:
+        if service is None:
+            service = initialize_service()
+        is_deleted = service.delete_account(name=account_name)
+
+        if is_deleted:
+            return {
+                "status": "success",
+                "deleted": True,
+                "message": "Account successfully deleted",
+            }
+        else:
+            return {
+                "status": "error",
+                "deleted": False,
+                "error": "Account name not found or could not be deleted",
+            }
+
+    except Exception as e:
+        logger.error(f"Failed to delete account: {e}")
+        return {"status": "error", "deleted": False, "error": str(e)}
+
+
+@with_sync
+async def list_saved_account() -> dict[str, Any]:
+    """
+    List all IBM Quantum accounts saved on disk.
+
+    Retrieves account information from the local configuration file without requiring authentication.
+    Useful for checking which accounts are available before initializing
+    the service.
+
+    Returns:
+        Dictionary containing account list status:
+        - On success with accounts: {"status": "success", "accounts": [list of account dicts]}
+          Each account dict contains: name, channel, and other metadata
+        - On success with no accounts: {"status": "success", "accounts": "No accounts found"}
+        - On error: {"status": "error", "error": error_message}
+    """
+    global service
+
+    try:
+        if service is None:
+            service = initialize_service()
+        accounts_list = service.saved_accounts()
+        if len(accounts_list) > 0:
+            return {"status": "success", "accounts": accounts_list}
+        else:
+            return {"status": "success", "accounts": "No accounts found"}
```

Comment:
For consistency with other functions that return lists, this should be

```suggestion
            return {"status": "success", "accounts": [], "message": "No accounts found"}
```